lib/hanami/cli.rb

```diff
@@ -131,6 +131,17 @@ def routes
       end
     end
 
+    desc 'secret', 'Prints a secure hash'
+    long_desc <<-EOS
+    `hanami secret` will generate a secure key to be used by the developer as they want, e.g. cookie sessions.
+    EOS
+    def secret(length = '64')
+      require 'securerandom'
+      # must divide by 2 because SecureRandom.hex multiplies it by 2
+      divided_length = length.to_i / 2
```

Comment:
Well, I could update the code to something like: ``SecureRandom.hex(length)[0..(number_of_bytes - 1)].length``. (I will not write all in a single command like here)